### PR TITLE
Content changes to support docs

### PR DESCRIPTION
**Pull request by bethwalton**
Small changes for clarity and consistency. Includes changing references to GovWifi admin and making headings and link text more descriptive.

### CODE REVIEW

File: source/documentation/introduction.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Offer GovWifi
 
-GovWifi is a wifi authentication service using RADIUS server technology allowing staff and visitors to use a single user login to connect to guest wifi across the public sector. 
+GovWifi is a wifi authentication service using RADIUS server technology. It allows staff and visitors to use a single username and password to connect to guest wifi across the public sector.
 
 If you manage IT services in a public sector organisation and would like to make GovWifi available in your buildings, use this technical documentation to find out how to:
 
 - set up GovWifi for your public sector organisation
 - manage the GovWifi connection in your buildings
```
Comment (joelanman) [inline source/documentation/introduction.md:8]: 'connection' feels a bit odd here, maybe just 'manage GovWifi in your buildings'

File: source/documentation/manage.md [hunk 2]
```diff
@@ -10,36 +10,36 @@ After you’ve successfully deployed GovWifi, consider removing less secure gues
 
 ## View and update your site configuration
 
-You can view and update your GovWifi configuration in the [GovWifi admin platform](https://admin.wifi.service.gov.uk/)
+You can [view and update your configuration in your GovWifi admin account](https://admin.wifi.service.gov.uk/)
```
Comment (joelanman) [inline source/documentation/manage.md:13]: would it be closer to our style guide to have
> You can [view and update your configuration in GovWifi admin](https://admin.wifi.service.gov.uk/)

File: source/documentation/manage.md [hunk 3]
```diff
@@ -10,36 +10,36 @@ After you’ve successfully deployed GovWifi, consider removing less secure gues
...
-## Removing users from GovWifi
+## Remove users from GovWifi if necessary
 
 If you find someone breaching your acceptable use policy, you can remove their access from your local network.
 
 We will remove users for breaching GovWifi’s [terms of service](https://www.gov.uk/government/publications/terms-and-conditions-for-connecting-to-govwifi/terms-and-conditions-for-connecting-to-govwifi). This includes, but is not limited to, illegal activity and breaching the [Civil Service Code](https://www.gov.uk/government/publications/civil-service-code/the-civil-service-code).
 
 
-### Reporting abuse
+### Report abuse
```
Comment (joelanman) [inline source/documentation/manage.md:28]: does this even need a heading? Seems to fit well under 'Remove users from GovWifi if necessary'

File: source/documentation/manage.md [hunk 4]
```diff
@@ -10,36 +10,36 @@ After you’ve successfully deployed GovWifi, consider removing less secure gues
...
 
-If you believe a user is breaching the GovWifi terms of service, [please contact us](https://admin.wifi.service.gov.uk/help).
+If you believe a user is breaching the GovWifi terms of service, [contact us](https://admin.wifi.service.gov.uk/help).
 
 
-## Logging traffic
+## Log traffic
```
Comment (joelanman) [inline source/documentation/manage.md:33]: I wonder if this heading should be an action. The part that pertains to GovWifi is about requesting logs. So maybe just 'Logs' or 'Traffic logs' would be a better heading

File: source/documentation/set_up.md [hunk 5]
```diff
@@ -4,31 +4,29 @@ Follow these steps to set up and make GovWifi available in your organisation.
...
-[GovWifi admin portal](https://admin.wifi.service.gov.uk/ips).
+Create a [network address translation (NAT) rule](https://wiki.untangle.com/index.php/NAT_Rules) if one does not already exist.
 
-Create a [network address translation (NAT) rule](https://wiki.untangle.com/index.php/NAT_Rules) if one does not already exist. This allows your wifi controller or access points to connect to the internet using the IP addresses you set up in the [admin portal](https://admin.wifi.service.gov.uk/ips).
+This will allow your wifi controller or access points to connect to the internet using the IP addresses you'll add to your GovWifi admin account.
 
 ### Set up your public IP addresses
 
-You should add your IP address or addresses to your admin portal profile.
+Add your IP address or addresses to your [GovWifi admin account](https://admin.wifi.service.gov.uk/ips).
```
Comment (joelanman) [inline source/documentation/set_up.md:27]: 'address or addresses' could just be 'addresses' I think

File: source/documentation/set_up.md [hunk 6]
```diff
@@ -4,31 +4,29 @@ Follow these steps to set up and make GovWifi available in your organisation.
...
 
-You can find details of the Remote Authentication Dial-In User Service (RADIUS) servers on the admin platform. You must connect to at least one (RADIUS) server. You can configure your authenticators to connect to more than one (RADIUS) server. These extra servers are known as ‘backups’ or ‘failovers’.
+You can find details of the Remote Authentication Dial-In User Service (RADIUS) servers in your GovWifi admin account. You must connect to at least one RADIUS server. You can configure your authenticators to connect to more than one RADIUS server. These extra servers are known as ‘backups’ or ‘failovers’.
```
Comment (joelanman) [inline source/documentation/set_up.md:29]: do we still need brackets around the first RADIUS?

File: source/documentation/set_up.md [hunk 7]
```diff
@@ -40,27 +38,25 @@ We recommend connecting to all GovWifi RADIUS servers, so you're protected from
...
 ### Remotely deploy GovWifi profiles to managed devices
 
-You must deploy profiles to your managed devices for automatic certificate checking. You could use mobile device 
-management solutions or operating system mechanisms for this. For example:
+You must deploy profiles to your managed devices for automatic certificate checking. You could use mobile device
+management solutions or operating system mechanisms for this. For example, you can deploy profiles on:
 
-* [deploy profiles on Windows devices using Group Policy](https://en.wikipedia.org/wiki/Group_Policy)
-* [deploy profiles on Apple devices using the Profile Manager](https://support.apple.com/en-gb/profile-manager)
+* Windows devices using [Group Policy](https://en.wikipedia.org/wiki/Group_Policy)
```
Comment (joelanman) [inline source/documentation/set_up.md:53]: I wonder if the link was the full text on both bullets, that would be slightly more accessible?

File: source/documentation/set_up.md [hunk 8]
```diff
@@ -103,12 +99,12 @@ netsh wlan set profileorder name="GovWifi" interface="Wi-Fi" priority=1
 
 ## Advertise GovWifi in your organisation
 
-You should advertise GovWifi in the locations you’ll roll it out.
+To make sure your users know they can use GovWifi, you should advertise the service.
```
Comment (joelanman) [inline source/documentation/set_up.md:102]: should > need to?

File: source/documentation/set_up.md [hunk 9]
```diff
@@ -103,12 +99,12 @@ netsh wlan set profileorder name="GovWifi" interface="Wi-Fi" priority=1
...
 
-[Download a poster](https://admin.wifi.service.gov.uk/setup_instructions/poster) (you might need to sign in) with information on how end users can connect to GovWifi in your organisation.
+You can [use our poster](https://admin.wifi.service.gov.uk/setup_instructions/poster) (you might need to sign in to your admin account). It includes information on how your users can connect to GovWifi in your buildings.
```
Comment (joelanman) [inline source/documentation/set_up.md:104]: do we need 'in your buildings'?

File: source/documentation/troubleshooting.md [hunk 10]
```diff
@@ -1,19 +1,23 @@
 # Support
 
-The Government Digital Service supports GovWifi network administrators on problems with the central authentication service.
+We support GovWifi administrators on problems with the central authentication service.
 
-End-user support for GovWifi is the responsibility of the relevant IT support helpdesks of public sector organisations offering GovWifi. They will be able to check user devices and also ensure the local network is working.
+Your organisation is responsible for helping your users connect to GovWifi. This is because you can check for problems with their devices and your local network.
 
-## If an individual user in your organisation is having trouble connecting to GovWifi
+## If an individual user is having trouble connecting
```
Comment (joelanman) [inline source/documentation/troubleshooting.md:7]: an individual user > someone ?

**Review by freesteph — APPROVED** [review]

**Pull request merged**